tklock.c

```diff
@@ -341,6 +427,10 @@ static gboolean generic_event_control(const gchar *const file,
  */
 static gboolean ts_event_control(gboolean enable)
 {
+	if (generic_x11_event_control(enable)) {
+		return TRUE;
+	}
```

Comment:
Curly braces here differ from the code style above for single command in an if statement.

Reply:
Will fix